### PR TITLE
Adding damage visuals for directional windows

### PR DESCRIPTION
**Pull request by Darkenson**
Added a crack overlay for directional windows. Added directional window crack components to windows, reinforced windows, plasma windows and reinforced plasma windows.

## About the PR

Changed the entity components for directional windows, reinforced windows, plasma windows and reinforced plasma windows.

## Why / Balance

Directional windows didn't have any cracks when damaged, while regular windows did. Should make station maintenance easier.

## Technical details

Added damage overlay components in the windows, reinforced windows, plasma windows and reinforced plasma windows entity YAML files. Added a new directional crack sprite made by editing the regular one.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, 
https://github.com/space-wizards/space-station-14/assets/4154895/331dea77-e7c6-45d9-9315-6fac46029912

## Breaking changes

**Changelog**

:cl: Darkie
- add: Added directional window cracks when damaged.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
I was gonna say stencil but this is probably better art-wise

File: Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation at commit e06b82a7f4b2b09216fb28fd384c95a2e1dc50e5 and modified by Darkie",
```
Comment (metalgearsloth) [inline Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json:8]: Please don't copy-paste copyright, use the commit you actually got it from as this is 2 years old

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: Fixed! I think. 
This is the commit I got them from: https://github.com/vulppine/space-station-14/commit/f57e8ec6b9b4b72ef56c8146be0bc159ed2691ee

Comment (metalgearsloth) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: Did flipp make these sprites or did they get it from somewhere?

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: I'm not sure, I can't find the original source from the original copyright. I took the cracks sprites that were already there for regular windows and cropped them in order to make them work. Maybe the original copyright for the crack sprite is wrong to begin with.

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: Ah, I think I found the original source: [e34888373a9436af3f09672c2d92757721b5a94c](https://github.com/vulppine/space-station-14/commit/e34888373a9436af3f09672c2d92757721b5a94c) . This is when the crack files seem to have been created.

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: Looking at that commit really lags out my browser, it's massive.

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: And digging deeper, looks like originally made here https://github.com/SweptWasTaken/space-station-14/commit/d59cc139b275fde1e6c1328c7862aab98e9cb2f6 by Vera.

Comment (Darkenson) [reply Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json]: And that's where the taken from TGSstation copyright is also from.
So... what do?

**Pull request merged**